blocks/rich-text/format-toolbar/index.js

```diff
@@ -19,21 +19,25 @@ const FORMATTING_CONTROLS = [
 	{
 		icon: 'editor-bold',
 		title: __( 'Bold' ),
+		shortcut: '⌘B',
```

Comment:
I'm concerned about the hard-coding of the modifier key as `⌘`, which is meaningless to non-macOS users. IMO, this is a blocker.

There's precedent of cross-platform handling:

https://github.com/WordPress/gutenberg/blob/ca593c5025fcceee02712a7217151910817e238f/edit-post/keyboard-shortcuts.js#L1-L2